### PR TITLE
Fix #2823: Added label for ProfileListActivity for correct talkback output

### LINKED ISSUE #2823: [A11y] Add label for ProfileListActivity
## Task
Introduce label for `ProfileListActivity` as `Profile List`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by AnubhavPabby**
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by AnubhavPabby** [comment]
@rt4914 @BenHenning PTAL

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/misc.xml:9]: Revert changes from this file.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -96,7 +96,8 @@
     <activity android:name=".app.settings.profile.ProfileEditActivity" />
     <activity
       android:name=".app.settings.profile.ProfileListActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/profile_list_activity_title"/>
```
Comment (rt4914) [inline app/src/main/AndroidManifest.xml:100]: Add space before `/>`.

Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

File: app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListActivityTest.kt [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
+  // correct string when it's read out.
+  @Test
+  fun testProfileListActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(createProfileListActivityIntent())
+    val title = activityTestRule.activity.title
+
+    // Verify that the activity label is correct as a proxy to verify TalkBack will announce the
+    // correct string when it's read out.
+    Truth.assertThat(title).isEqualTo(context.getString(R.string.profile_list_activity_title))
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListActivityTest.kt:87]: Use direct import `assertThat(title).isEqualTo(context.getString(R.string.profile_list_activity_title))`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Did quick review. Suggested few changes.

**Comment by AnubhavPabby** [comment]
@rt4914 @BenHenning PTAL

> Did quick review. Suggested few changes.

Ok I would be doing the following changes and resubmit it

File: .idea/misc.xml (hunk 1 above)
Comment (AnubhavPabby) [reply .idea/misc.xml:9]: How can I revert the changes now from this file?

**Comment by prayutsu** [comment]
@AnubhavPabby  A  few quick tips for making a good PR - 
1. The commit messages should never include keywords like `Fixes #bugNum` because it leads to unnecessary mentions of the commit in the issue description.
2. Always add `Fixes #bugNum` in the PR explanation followed by a short explanation.1
3. Only check the checkBoxes in the PR description when you have made the changes as insrcuted.

**Comment by prayutsu** [comment]
@AnubhavPabby Any updates on this?

**Comment by FareesHussain** [comment]
@AnubhavPabby Are you working on this?

File: .idea/misc.xml (hunk 1 above)
Comment (FareesHussain) [reply .idea/misc.xml:9]: You can refer [changes in misc.xml](https://github.com/oppia/oppia-android/pull/2921/files#diff-7ea1cd7bd5d16299a0abd33128e8d0a8bc04146f6b00c07bcd3cc0aad813c229) and make sure the there are no changes left

**Comment by anandwana001** [comment]
Closing this PR as no further update.

**Pull request closed without merging**